scripts/exaqm_bias_correction_pm25.sh

```diff
@@ -209,8 +211,50 @@ fi
 cpreq ${DATA}/out/pm25/${yyyy}/*nc ${DATA}/data/bcdata.${yyyymm}/interpolated/pm25/${yyyy}
 
 if [ "${DO_AQM_SAVE_AIRNOW_HIST}" = "TRUE" ]; then
-mkdir -p  ${COMOUTbicor}/bcdata.${yyyymm}/interpolated/pm25/${yyyy}
-cpreq ${DATA}/out/pm25/${yyyy}/*nc ${COMOUTbicor}/bcdata.${yyyymm}/interpolated/pm25/${yyyy}
+
+   # NetCDF files
+   for i in {1..3}; do
+      yyyymm_m="yyyymm_m${i}"
+      yyyy_m="yyyy_m${i}"
+      PDYm="PDYm${i}"
+
+      target_dir="${COMOUTbicor}/bcdata.${!yyyymm_m}/airnow/netcdf/${!yyyy_m}/${!PDYm}"
+
+      if [ ! -d "$target_dir" ]; then
+           mkdir -p "$target_dir"
+      fi
+
+      # Check if the file exists before attempting to copy it
+      if [ -s "${DATA}/data/bcdata.${!yyyymm_m}/airnow/netcdf/${!yyyy_m}/${!PDYm}/HourlyAQObs.${!PDYm}.nc" ]; then
+          cp "${DATA}/data/bcdata.${!yyyymm_m}/airnow/netcdf/${!yyyy_m}/${!PDYm}/HourlyAQObs.${!PDYm}.nc" "${COMOUTbicor}/bcdata.${!yyyymm_m}/airnow/netcdf/${!yyyy_m}/${!PDYm}"
```

Comment:
cp/cpreq?

Reply:
It is the same as the operation job, exaqm_bias_correction_pm25.sh at /lfs/h1/ops/prod
/packages/aqm.v7.0.7/scripts/.